src/components/controls/gamepad-controls.js

```diff
@@ -0,0 +1,110 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
Hmm, I imagine proper Cardboard experiences would use hotspot navigation and/or reserve the touch controls for clicking on stuff.

Though you could say something similar for mousedrag, but that's sort of the bare minimum so people can preview scenes without an HMD. Scenes that actually intend to use the mouse alongside an HMD though, those scenes won't work well on flat experiences. Tough compromises.